src/main/java/seedu/dictionote/ui/MainWindow.java

```diff
@@ -368,7 +371,7 @@ private void setPanelVisibility(Pane pane, boolean visible) {
     }
 
     /**
-     * Setup display panel.
+     * Setups display panel.
```

Comment:
Shouldn't this be *"Set**s up**"* instead?

Reply:
fixed